Formula/tcpreplay.rb

```diff
@@ -12,23 +12,19 @@ class Tcpreplay < Formula
     sha256 "ab0379428462fbdc2653feae2bcc404cf6b6d2129ddf0461019c53794ce87e4f" => :mavericks
   end
 
-  depends_on "libdnet" => :recommended
+  depends_on :xcode
```

Comment:
Really? I thought `MacOS.sdk_path` was just `nil` when Xcode wasn't around, so it became effectively `--with-libpcap=/usr`. Was I sniffing something funny the day I thought that?